### ISSUE TITLE
Maximum length?

### ISSUE THREAD

**Issue by peteruhrig** [pettarin/forced-alignment-tools#4]
Dear Alberto,

thanks a lot for this great list! Even though you do not list this property in the table, I would be interested to hear if you have any insights as to the maximum length of recordings and transcripts for the various tools. At the [Distributed Little Red Hen Lab](http://www.redhenlab.org), we need to align one-hour recordings with their transcripts. This works great with Gentle for English, but for instance the Montreal Forced Aligner is only happy with relatively short passages. I'd be grateful for any pointers you have.

Best regards,
Peter

**Comment by pettarin** [comment]
I agree on the usefulness of a "maximum length" column. On the other hand, it is difficult to be objective there: the "max processable length" might depend on multiple factors (like: RAM available, acoustic/linguistic models being used, etc.) so it would be difficult to give a fair assessment without spending a considerable amount of time or without setting up a rigorous test environment.
Also, some algorithms might trade execution time, space/memory required and alignment accuracy.

Probably a qualitative/approximate description ("5 minutes" vs "10 hours" with 8GB RAM) would suffice.

**Comment by pettarin** [comment]
Ah, for clarity: I do not have a data point for each of the aligners currently in the table. In fact, I wiped out the linux laptop where I had most of them installed...
